### ISSUE TITLE
I can't seem to get the audio/portrait to work

### ISSUE THREAD

**Issue by xkurosawax** [Arrenton/KingdomHeartsPlugin#35]
I am having an issue getting both my portrait and sound working. I placed the directory in the correct pathway and I am not seeing any results. The pathway I have is C:\Users\danie\Pictures\Ffxiv\image.png and nothing is happening when I hit the set button and save. Is there something I am missing? Please let me know because this is a cool mod! :D

**Comment by xkurosawax** [comment]
):

**Comment by Arrenton** [comment]
Sorry, I have been really busy as of late.

Normally when you click set for the portrait, it should pop up.
Did you set a portrait for all of them? The path looks to be correct.

**Comment by xkurosawax** [comment]
I'll send you a bunch of screenshots of how I have it.

**Comment by xkurosawax** [comment]
Okay so, here's what I have set so far.

1.) (https://drive.google.com/file/d/1paSFnQCa5MXEffziJgetOem6BNe1BCgj/view?usp=sharing)
 - This one is the file location where I place the image at.

2.)  (https://drive.google.com/file/d/1UpKL79utC4XQvykUS5-F6chXxhv5cRGy/view?usp=sharing)
- This one is just one file location I picked on the settings and not all because I was seeing if maybe trying one slot will work.

3.) (https://drive.google.com/file/d/1XBiT4-w9OVLxsU73AP7VppMQ_LSpT5yU/view?usp=sharing)
- This one I have it all selected for a file location with the same image as well. I am not sure if it needs to be all different images in order for it to work. I doubt it because it should just take whatever image I place it in. 

I have also tried this with the audio as well and the test button for me doesn't work to hear the custom audio sound. I have tried to let enemies get me to low hp and that still doesn't work.  Furthermore, I have tried resetting the plugins and uninstalling all the features but that still gives me the same problem of not working. Please let me know if you have any solutions because I have tried uninstalling/reinstalling the plugin, and changing file locations from SSD to Hard drive and both still have the same issue.

**Comment by Arrenton** [comment]
I can't view the images. Says access denied.

**Comment by xkurosawax** [comment]
I just updated it so let me know if it works and you are able to see my screenshots.

**Comment by xkurosawax** [comment]
I know the text might be a little small so let me know if you want me to zoom it in for you and you can see the file path better.

**Comment by Arrenton** [comment]
I immediately found your problem in the first image.
Your file is named image.png.jpg lol

**Comment by xkurosawax** [comment]
This whole time I just needed to change the file type to png. I guess you only set it to png images only because I have tried doing image.jpg and that wouldn't take it. But now, when I convert the file from jpg to png it works! Ahhh I should have known to change the file type to .png.  I got the sound to work as well so everything is good!!! Thank you so much for your help Arrenton and thank you for creating this wonderful mod! I love it!!!! 👍 :D
